cloud/microsoft/office365/exchange/mode/emailactivity.pm

```diff
@@ -24,6 +24,7 @@ use base qw(centreon::plugins::templates::counter);
 
 use strict;
 use warnings;
+use Time::Local;
```

Comment:
I'm not sure about this. 

I'm not very familiar with this lib and the timegm() function ... My understanding is that we will always get UTC instead of local TZ? Can't we imagine give the ability for the user to choose the TZ?

Reply:
Sorry for my late answer on this @Sims24.

You're right, `timegm` will generate a UTC timestamp.
Broker itself uses UTC timestamps in the events.
We then must use UTC in the `perfdate` timestamp we generate (the _Report Refresh Date_) to replace the default one (the "now" timestamp).

Reply:
Sorry for my late answer on this @Sims24.

You're right, `timegm` will generate a UTC timestamp.
And after digging into this again, timestamps used in the events (by the broker etc...) are local timestamps.
We then must use local time in the `perfdate` timestamp we generate (the _Report Refresh Date_) to replace the default one (the "now" timestamp).
Let me push a fix accordingly.

Reply:
Fix pushed, thank you for this 👍